src/lib/substituteResponsiveAtRules.js

```diff
@@ -23,13 +24,27 @@ export default function(config) {
       })
 
       mediaQuery.append(
-        rules.map(rule => {
+        responsiveRules.map(rule => {
           const cloned = rule.clone()
           cloned.selectors = _.map(rule.selectors, selector => `.${screen}\\:${selector.slice(1)}`)
           return cloned
         })
       )
-      css.append(mediaQuery)
+
+
+      finalRules.push(mediaQuery)
     })
+
+    let includesScreenUtilitiesExplicitly = false
+    css.walkAtRules('tailwind', atRule => {
+      if (atRule.params === 'screen-utilities') {
```

Comment:
Do you know if it's possible to extend the "css" variable somehow to provide new methods?

```css.walkTailwindRules('screen-utilities')``` or ```css.hasTailwindRules('screen-utilities')``` would be handy in many places. Saw this kind of checks quite a few times.